### PR TITLE
Fix for #146

### PR DESCRIPTION
**Pull request by azawlocki**
The option `--public-address` is now only valid for `tests/imunes/node.py`, not for `gnr/node.py`. Specifying `--public-address` results in bypassing `Node's` `collect_network_info`  (stun is not used) and setting `Node.pub_addr` and `Node.priv_addr` to the specified value.

### CODE REVIEW

File: golem/client.py [hunk 1]
```diff
@@ -41,6 +41,9 @@ def create_client(**config_overrides):
     config_desc.init_from_app_config(app_config)
 
     for key, val in config_overrides.iteritems():
+        # For safety we only allow here overriding properties that are
+        # fields of ClientConfigDescriptor. Hence the getattr() here:
+        getattr(config_desc, key)
```
Comment (chfast) [inline golem/client.py]: There is also `hasattr()` builtin function. It might be more explicit.

File: tests/imunes/test_imunes_node.py [hunk 2]
```diff
@@ -4,6 +4,7 @@
 from golem.core.simpleenv import SimpleEnv
 from gnr.renderingenvironment import BlenderEnvironment
 from golem.tools.testwithappconfig import TestWithAppConfig
+# from golem.client import create_client
```
Comment (chfast) [inline tests/imunes/test_imunes_node.py]: Remove.

File: golem/client.py (hunk 1 above)
Comment (azawlocki) [inline golem/client.py]: You're right, never write in comments what you can write in code. Fixed in 2eff0831aad71e868cf53f0a6654bba19bb77f80

File: tests/imunes/test_imunes_node.py (hunk 2 above)
Comment (azawlocki) [inline tests/imunes/test_imunes_node.py]: Removed.

**Pull request merged**